### DISCUSSION TITLE
How to edit vue components in nova 4 ?

### DISCUSSION THREAD

**Discussion by SupchickCode** [laravel/nova-issues#4054] [Q&A]
For example, how you can change the footer ?

In nova 3 you could publish blade files and change, but what about vue ?

Or you need to edit the component in the vendor folder and then publish it, but it's looks weird

**Comment by wdelfuego** [comment]
This is not a general solution to editing the view files but I’ve created a small package that at least allows you to customize the content of the footer: https://github.com/wdelfuego/nova4-customizable-footer

It works by updating the DOM after Nova’s been loaded; a general solution to replace/edit view templates before they’re being sent to the client would be awesome.

**Reply by SupchickCode** [reply]
thx u =)
